octopoes/octopoes/models/__init__.py

```diff
@@ -241,7 +241,10 @@ def format_id_short(id_: str) -> str:
 class PrimaryKeyToken(RootModel):
     root: Dict[str, Union[str, PrimaryKeyToken]]
 
-    def __getattr__(self, item) -> Union[str, PrimaryKeyToken]:
+    def __getattr__(self, item):
```

Comment:
This would still return a str or PrimaryKeyToken, correct?

Reply:
Correct! I accidentally left out the type hints. Updated.